packages/web-api/src/WebClient.ts

```diff
@@ -130,6 +136,12 @@ export type PageAccumulator<R extends PageReducer> = R extends (
   ? A
   : never;
 
+export type RequestConfig = InternalAxiosRequestConfig;
```

Comment:
I was not sure what the best practice was here. My thinking was that since `axios` is not set as a peer dependency of the `web-api` package, I did not want clients to import an interface from `axios`, but rather keep the imports contained to the WebClient, as shown in the tests [here](https://github.com/slackapi/node-slack-sdk/pull/2076/files#diff-e07240a5e03c68dd380c2aa81e3b829fd336cadb30770b1c0a8fa19af95e24dcR7)

Reply:
Nice, OK, makes sense / sounds good. Let's drop a JSDoc above that to capture this sentiment; most IDEs and our reference generation support linking to pre-existing constructs within JSDoc. Maybe we can add something like the following to help create a papertrail between these various constructs:

```javascript
/**
 * An alias to {@link https://github.com/axios/axios/blob/v1.x/index.d.ts#L367 axios' `InternalAxiosRequestConfig`} object, which is the main parameter type provided to axios interceptors and adapters.
 */
```

Or some such... that will get properly parsed by our documentation generator as well as most IDEs / LSP integrations.

Reply:
JSDoc has been added